packages/ifpack/src/Ifpack_Hypre.cpp

```diff
@@ -79,9 +79,13 @@ Ifpack_Hypre::Ifpack_Hypre(Epetra_RowMatrix* A):
   IsSolverSetup_[0] = false;
   IsPrecondSetup_[0] = false;
   MPI_Comm comm = GetMpiComm();
+  // Hypre expects GIDs that are:
+  //  - contiguous locally, in the sense that GID[i+1] = GID[i]+1
+  //  - contiguous across ranks, in the sense that GID_rank_k[0] = GID_rank_(k-1)[-1]+1
+  // Here we call this property "nice".
```

Comment:
Haha, yeah, contiguous is used in many different ways and I wasn't sure what your convention was.  Will update.

Or should it be LinearMap_ instead of SimpleMap_ or NiceMap_ or GloballyContiguousMap_?

Reply:
> Or should it be LinearMap_ instead of SimpleMap_ or NiceMap_ or GloballyContiguousMap_?

I would prefer `GloballyContiguousRowMap_`.